### PR TITLE
Spaceport culture review

### PR DESCRIPTION
**Pull request by Amazinite**
The mug outfit doesn't have an image right now, but we can fill that in later. The mug's very existence as an outfit is funny enough without the image. Used the new conversation `action` functionality to gift the mug to the player and deduct credits from their account if they choose to buy it. Branch if the player doesn't even have enough credits for it.

### CODE REVIEW

**Comment by Amazinite** [comment]
All good here. I only have a single comment, posted as a review over on the PR, about the description of a human and Hai pair as "uncommon."

File: data/human/spaceport conversations.txt [hunk 1]
```diff
@@ -66,40 +31,56 @@ mission "ZZCU Sad Gamblers -S"
...
-# If anyone wants to add receipt of an actual mug outfit feel free:
-# Amazinite: Let's give the player a mug outfit and take 50 credits from them by making this an invisible mission where on accept you get the outfit and lose the credits, then automatically fail the mission.
-# "Mug: Hand to hand outfit. +0.1 boarding attack. You can throw it in a pinch, you guess." - @Galaucus
+			
+			label broke
+			`	Curious, you turn over your own mug, and when you spot the 50 credit price tag your heart sinks. You could never afford such an expensive mug. You put it back and walk away while pondering your financial situation.`
+				decline
 
-mission "ZZCU Trash Fire Crops"
+outfit "Mug"
```
Comment (Amazinite) [inline data/human/spaceport conversations.txt:73]: I suddenly had the idea of having a way to get seven mugs from seven different such missions from different, including alien, regions of the galaxy, unlocking a secret line of missions that's entirely designed as a joke.

File: data/human/spaceport conversations.txt [hunk 2]
```diff
@@ -66,40 +31,56 @@ mission "ZZCU Sad Gamblers -S"
...
 
 
-#               Republic Only               #
 
-mission "ZZCU Gift Store Interaction"
+mission "Gift Store Interaction"
 	minor
 	source
 		government "Republic"
-		attributes "deep"
```
Comment (MasterOfGrey) [inline data/human/spaceport conversations.txt]: This was in the Deep deliberately.

**Comment by MasterOfGrey** [comment]
You couldn't just let me have "Culture Conversations" and mission prefixes could ya? 

Edit: Also checks don't all succeed at present.

File: data/human/spaceport conversations.txt [hunk 3]
```diff
@@ -66,40 +31,56 @@ mission "ZZCU Sad Gamblers -S"
...
+		attributes "tourism"
 		not attributes "station"
 	to offer
 		random < 1
 	on offer
 		conversation
-			`As you wander the spaceport with a snack, waiting for your ship to be refueled, you pass a local souvenir shop and poke your head in to kill a little time. You see a decent looking mug with the text "<source>" printed underneath an attractive, stylized representation of the spaceport you are presently standing in and pick it up to see how it feels. The handle is solid, not too small for your fingers, and the lip of it looks like it would be comfortable to drink out of.`
-			`	Two young adults wander into the aisle just a couple of meters away. They're clearly from abroad judging by their conversation occurring in some local dialect which is neither local nor one you understand. They're cooing over an item they've found, but when one flips it over to look at the price tag, there is an immediate exchange of looks and expressions that does not require translation. The item ends up back on the shelf with a giggle and a head-shake.`
```
Comment (MasterOfGrey) [inline data/human/spaceport conversations.txt:87]: This pertains to this: `* Changed the talk of dialects to accents. Dialects are language variants; even if they were speaking a different dialect of the same language you're speaking, you should be able to understand them. If they're speaking a different language, then talking about the dialect doesn't matter, and talking about the different language would make more sense. But the distinction between accents in different region of space has been used before in other missions, hence the switch to highlighting the accents.`

The entire point of this bit was to capture that you explicitly did not understand any of the spoken words, but nevertheless had a clear idea of exactly what was going on. Going with accents I feel takes that the opposite direction and is incongruous with the following sentence of relying on looks and expressions.

The entry for dialects also explicitly includes the comment that, "Some dialects of a language are not mutually intelligible in spoken form."

**Comment by Amazinite** [comment]
> Edit: Also checks don't all succeed at present.

Pretty sure that's just a matter of your branch being very out of date.

**Comment by MasterOfGrey** [comment]
> Pretty sure that's just a matter of your branch being very out of date.

Aight, I have pushed an update from master.

**Comment by Amazinite** [comment]
You chose the wrong master branch. :)

**Comment by MasterOfGrey** [comment]
Ah I see you've done it. Ok.

Any response to my other comments?

**Comment by Amazinite** [comment]
Does the change I made in the most recent commit look good?

**Comment by MasterOfGrey** [comment]
> Does the change I made in the most recent commit look good?

It does.

I still don't see why "Culture Conversations" wasn't acceptable?
Or why it was necessary to take out mission prefixes when they made no actual difference to you but did actually represent some value to some other people?

It feels spiteful, even though I'm sure it's probably not.

**Comment by Amazinite** [comment]
I'm pretty sure I've explained my reasoning for both in the past, however long ago that might have been now.

**Comment by MasterOfGrey** [comment]
> I'm pretty sure I've explained my reasoning for both in the past, however long ago that might have been now.

For the file name we actually chatted on discord and you seemed ok with that one at the time.

For the prefixes you explained why you didn't think it was necessary, but never what value you personally saw in explicitly taking away the value that other people saw in it.

**Pull request merged**